src/plugins/subsystems/drm/drm.sh

```diff
@@ -273,21 +274,28 @@ function get_available_connectors()
       target_label='local'
       ;;
     3) # REMOTE TARGET
-      find_conn_cmd="find ${SYSFS_CLASS_DRM} -name 'card*'"
+      find_conn_cmd="find ${SYSFS_CLASS_DRM} -name 'card*-*' -exec printf '%s,' {} \; -exec cat {}/enabled \; -exec printf '\n' \;"
```

Comment:
This line and the one in the previous part of the `case` are very similar; to avoid duplication, put both of them in a single variable.

Reply:
In the localt target, should we use "eval" to execute the command that we store in the variable?

Reply:
Hi @FelipeAnibal. Sorry for parachuting into the conversation, but I may be able to help :sweat_smile: 

In the local target, FWIU, I think you can store the whole command inside `$()` inside a variable, say, `foo`, and execute it with `cmd_manager` like `cards_raw_list=$(cmd_manager 'SILENT' "$foo")`. Underneath, this function uses `eval`, but it is an abstraction layer that allow us to control its execution (the first argument `SILENT` tells the function to evaluate it in non-verbose mode). For more info in `cmd_manager` you can see its use throughout the project and look at its [implementation](https://github.com/kworkflow/kworkflow/blob/af4d2e3100bd33c8dd07c8dba2c42cd7af020c9b/src/lib/kwlib.sh#L70).